keyboards/sneakbox/aliceclone/keymaps/via/keymap.c

```diff
@@ -0,0 +1,46 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a license header, here? 

Additionally, VIA uses 4 layers by default, and all 4 layers need to be populated here, or the layers that explicitly defined will be populated with random data.